site/en/blog/fedcm-origin-trial/index.md

```diff
@@ -666,40 +624,6 @@ that must have the following properties:
      <td><code>clientId</code> (required)</td>
      <td>The RP's client identifier, issued by the IdP.</td>
   </tr>
-  <tr>
-     <td><code>hint</code> (optional)</td>
-     <td>A unique user ID that matches the one in the <a href="#accounts-list-endpoint">accounts list endpoint</a></td>
-  </tr>
-</table>
-
-This object provides the RP a method for users to [sign-in](#sign-into-rp),
-[sign-out](#signout-rp), or [revoke tokens provided by the
-IdP](#revoke-rp-access).
-
-#### Allow users to sign in {: #sign-into-rp}
-
-Call `FederatedCredential.login()` for both sign-up and sign-in dialogs. The
-browser handles these use cases differently depending on the existence of
-`approved_clients` in the response from [the accounts list
-endpoint](#accounts-list-endpoint). The browser will only display [the RP's
-privacy policy and terms of service](#client-metadata-endpoint) in the dialog if
-`approved_clients` isn't provided and the user hasn't previously signed up for
-the RP in this browser.
-
-```javascript
-const nonce = '15863786';
-const { id_token } = await credential.login({ nonce });
```

Comment:
The token can be retrieved from the credential object returned from `n.c.get()`.

Reply:
This is a removed section.